pyphare/pyphare/pharein/diagnostics.py

```
@@ -250,7 +250,7 @@ def to_dict(self):
 
 # ------------------------------------------------------------------------------
 class ElectromagDiagnostics(Diagnostics):
-    em_quantities = ["E", "B"]
+    em_quantities = ["E", "B", "B1", "B0", "divB"]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)